excitebike/src/test/java/com/codenjoy/dojo/excitebike/model/PlayersSpawnSystemParametrizedTest.java

```diff
@@ -0,0 +1,217 @@
+                                "  e  e \n" +
+                                " e  e  \n" +
+                                "  e  e \n" +
+                                " o  e  \n" +
+                                "■■■■■■■\n"
+                },
+                new Object[]{"shouldAddOneNewBike",
+                        1,
+                        "■■■■■■■" +
+                                "       " +
+                                "       " +
+                                "   o   " +
+                                "       " +
+                                "    o  " +
+                                "■■■■■■■",
+                        "■■■■■■■\n" +
+                                "       \n" +
+                                "       \n" +
+                                "   o   \n" +
+                                "  e    \n" +
+                                "    e  \n" +
+                                "■■■■■■■\n"
+                },
+                new Object[]{"shouldAddManyNewBikes",
+                        3,
+                        "■■■■■■■" +
+                                "       " +
+                                "     o " +
+                                " o     " +
+                                "      o" +
+                                "    o o" +
+                                "■■■■■■■",
+                        "■■■■■■■\n" +
+                                " e     \n" +
+                                "  e  o \n" +
+                                " e     \n" +
+                                "  e   e\n" +
+                                "    e e\n" +
+                                "■■■■■■■\n"
+                }
+        );
+    }
+
+    @Test
+    public void shouldSpawnPlayers() {
+        //given
+        MapParser mapParser = new MapParserImpl(init);
+
+        Dice dice = mock(Dice.class);
+        when(dice.next(anyInt())).thenReturn(5);
+        GameField field = new GameFieldImpl(mapParser, dice);
+        PrinterFactory factory = new PrinterFactoryImpl();
+
+        List<Game> games = new ArrayList<>();
+
+        if (mapParser.getBikes().isEmpty()) {
+            int playersNumber = StringUtils.countMatches(expected, BikeType.OTHER_BIKE.ch()) + 1;
+            for (int i = 0; i < playersNumber; i++) {
+                Game game = createNewGame(field, factory);
+                games.add(game);
+            }
+        } else {
+            for (Bike bike : mapParser.getBikes()) {
+                Game game = createNewGame(field, factory);
+                games.add(game);
+                ((Player) game.getPlayer()).setHero(bike);
+            }
+        }
+
+        //when
+        for (int i = 0; i < newPlayerNumberAfterInit; i++) {
+            Game game = createNewGame(field, factory);
+            games.add(game);
+        }
+        String res = (String) games.get(0).getBoardAsString();
+
+        //then
+        assertEquals(expected, res);
```

Comment:
Let's use hamcrest matcher assertThat instead, it shows really more explanatory failure message